pkg/server/alert/alert_notification.go

```diff
@@ -219,6 +220,34 @@ func (a *AlertService) TestNotification(ctx context.Context, req *alert.TestNoti
 	return &empty.Empty{}, nil
 }
 
+func (a *AlertService) RequestAuthzNotification(ctx context.Context, req *alert.RequestAuthzNotificationRequest) (*empty.Empty, error) {
+	if err := req.Validate(); err != nil {
+		return nil, err
+	}
+	notifications, err := a.repository.ListNotification(ctx, req.ProjectId, "slack", 0, time.Now().Unix())
+	sort.Slice((*notifications), func(i, j int) bool {
+		return (*notifications)[i].NotificationID < (*notifications)[j].NotificationID
+	})
+	notification := &(*notifications)[0]
```

Comment:
`(*notifications)[n]` という書き方が多いですが、`notifications[n]` では駄目ですか？（小カッコ入ります？）

Reply:
notificationsが *[]model.Notificationなのでできないようです。
以下の仕様書を見ると、array typeへのポインタの場合は小カッコがいらないのですが、slice typeへのポインタの場合は特に書いておらず、
> Otherwise a[x] is illegal.
とあるのでできないみたいです。
https://go.dev/ref/spec#Index_expressions